### ISSUE TITLE
Can you introduce the quadratic fitting

### ISSUE THREAD

**Issue by BeyondYourself** [hustvl/YOLOP#14]
I notice that the visualization of lane detection result has been post processed by quadratic fitting.
Whether  there is a  code implementation.
thx

**Comment by olivia23333** [comment]
Yes, I completed a post-process to fit the lane. I tried two methods, however the performance of the fitting is not so good. So I did not add it to current repo. If you need it, I will upload the code of post-process recently.

**Comment by BeyondYourself** [comment]
Sure,you can upload your methods fo us reference
